cfg/prod.js

```diff
@@ -5,8 +5,9 @@ var base = require('./base.js');
 
 
 module.exports = _.extend(base, {
+    mode: 'production',
     plugins: [
-        new webpack.optimize.DedupePlugin(),
+        // new webpack.optimize.DedupePlugin(),
```

Comment:
Dead code. Either add a comment why this is commented out, or remove the line entirely

Reply:
Webpack 4 doesn't know this plugin. 

> This option is redundant in webpack 2.

(https://github.com/webpack/docs/wiki/optimization)